TestApp/Startup.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using FluentValidation.AspNetCore;
+using TestApp.Model;
+using Microsoft.EntityFrameworkCore;
+using TestApp.Repositories;
+using Swashbuckle.AspNetCore.Swagger;
+
+namespace TestApp
+{
+    public class Startup
+    {
+        //public Startup(IConfiguration configuration)
+        //{
+        //    Configuration = configuration;
+        //}
+        public Startup(IHostingEnvironment env)
+        {
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(env.ContentRootPath)
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+                .AddEnvironmentVariables();
+            Configuration = builder.Build();
+        }
+
+        public IConfigurationRoot Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            
+            var connection = @"Server=(localdb)\mssqllocaldb;Database=TestAPIDB;Trusted_Connection=True;";
+            services.AddDbContext<TestAPIContext>(options => options.UseSqlServer(connection));
+            services.AddScoped(typeof(IDataAccess<Person, int>), typeof(DataAccessRepository));
+            services.AddMvc().AddFluentValidation()
+                .AddFluentValidation(fv => fv.RegisterValidatorsFromAssemblyContaining<PersonValidator>())
```

Comment:
Why is called two times? with only one time should be enought. If not we are doing twice assembly scan

Reply:
I will check it